VOM_Fortran/VOM-input/vom_namelist

```diff
@@ -11,7 +11,7 @@ i_toptf=0.0d0,          ! parameter to calculate adaptation of topt (range 0-1,
 i_toptstart=305.0d0,    ! start parameter for topt to calculate jmax(temp in K)
 i_rlratio=0.07d0,       ! leaf respiration parameter (rl=rlratio*assmax)
 i_mdtf=10000.0d0,       ! total dry mass of living tissues of trees per unit pc g/m^2
```

Comment:
perhaps this should be changed to kg m-2

Reply:
Yup, that's the culprit. Found that the VOM was using grams instead of kilograms. I didn't change any calculations, just the descriptions of the parameters in the commented part. But will change it to kg consistently.